www/js/lib/uiUtil.js

```diff
@@ -86,34 +86,73 @@ define(rqDef, function(settingsStore) {
      * @param {String} nodeAttribute The attribute to set to the URI
      * @param {Uint8Array} content The binary content to convert to a URI
      * @param {String} mimeType The MIME type of the content
-     * @param {Function} callback An optional function to call with the URI
+     * @param {Function} callback An optional function to call to start processing the next item
      */
     function feedNodeWithBlob(node, nodeAttribute, content, mimeType, callback) {
```

Comment:
The name of this function is now inaccurate, as it no longer inserts BLOBs. Should it be changed to `feedNodeWithDataURI`?

Reply:
You're right. I'm doing it